src/app/todo/components/todoAdding/todo-adding.component..ts

```diff
@@ -14,8 +14,8 @@ import { HttpClient } from '@angular/common/http';
 @Component({
```

Comment:
you created an AutoUnsub decorator but it doesn't seem to be used in the component